diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3b47ae19..18373422 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,9 +144,9 @@ message(STATUS "Setting skupper-router version to ${QPID_DISPATCH_VERSION}")
 ##
 find_library(rt_lib rt)
 
-find_package(Python 3.6.8 REQUIRED COMPONENTS Interpreter Development)
+find_package(Python 3.9 REQUIRED COMPONENTS Interpreter Development)
 find_package(Threads REQUIRED)
-find_package(Proton 0.39.0 REQUIRED COMPONENTS Core Proactor Tls)
+find_package(Proton 0.37.0 REQUIRED COMPONENTS Core Proactor Tls)
 message(STATUS "Found Proton: ${Proton_LIBRARIES} (found version \"${Proton_VERSION}\")" )
 
 # http2 support
